Makefile

```diff
@@ -120,21 +120,28 @@ ifneq ("$(wildcard $(GINKGO))","")
 GO_TEST="$(GINKGO)" -cover -output-dir=.
 endif
 
+##@ General
+
 .PHONY: all
 all: manager
 
-# Run tests
-.PHONY: test
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Comment:
```suggestion
	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-30s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

This just helps space out the help summary:

![image](https://github.com/cryostatio/cryostat-operator/assets/68053619/62cf7c78-fb3b-4ccf-8c21-8071bc37e625)

Reply:
Here is the previous view with only 15 spaces.

![image](https://github.com/cryostatio/cryostat-operator/assets/68053619/c3a16188-f769-4d74-9eb8-2b1a96559126)